check-diff.sh

```diff
@@ -69,10 +70,16 @@ function set_environment_variables {
 			DIFF_BASE=${DIFF_BASE:-$TRAVIS_COMMIT^}
 		fi
 		DIFF_HEAD=${DIFF_HEAD:-$TRAVIS_COMMIT}
+	fi
+
+	if [[ -n "${BITBUCKET_BRANCH+set}" ]]; then
```

Comment:
@westonruter Thanks for that explanation. That makes sense. The syntax above makes complete sense as well, I should have gone that route to begin with.

The statement `[[ -n "${BITBUCKET_BRANCH+set}" ]]` seemed to be working, but it could be on a newer version of Bash or could have adverse effects on Travis builds.

Things are updated and tested and they look to be working well on my end.